### DISCUSSION TITLE
Consulta de la practica 1

### DISCUSSION THREAD

**Discussion by renatocoello** [Launch-X-Latam/MisionFrontEnd#98] [Q&A]
Tengo una pregunta es sobre la practica 1, que es lo entregable:

- El wireframe 
- Los bocetos 

Por el momento nada que programar verdad

**Comment by doguedogue** [comment]
Así es, tal cuál lo dices 🚀🚀🚀

**Comment by maxkario** [comment]
Pienso que con los Wireframe basta, siempre y cuando este todo bien formulado y explicado el flujo de trabajo.
